oozie/src/main/java/org/apache/falcon/oozie/OozieOrchestrationWorkflowBuilder.java

```diff
@@ -219,6 +230,25 @@ protected void decorateWorkflow(WORKFLOWAPP wf, String name, String startAction)
         wf.getDecisionOrForkOrJoin().add(kill);
     }
 
+    protected void addPostProcessing(WORKFLOWAPP workflow, ACTION action) throws FalconException{
```

Comment:
As Falcon has the security context, you don't need to add HDFS server configs for post processing.